### PR TITLE
Responsive Web design: Milestone 1

### PR DESCRIPTION
**Pull request by Emmyn5600**
#### John & Emmy

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+<title>TNW</title>
+<link rel="stylesheet" href="reset.css"/>
+<link rel="stylesheet" href="style.css"/>
+<link rel="shortcut icon" type="image/jpg" href="./assets/favicon.png"/>
+<script src="https://kit.fontawesome.com/fd0eaf46fd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+<div class="navbar-relative-div">
```
Comment (bini-i) [inline index.html]: Please make sure to use ```header``` tag for the header

File: index.html [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+      <img class="latest-img" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1346411998544879617%2FQQiDnIXT.png&signature=6a474c1592a800091fe31b2d54a18545" alt="ringover-logo"/>
+      <div class="latest-post">
+        <div class="company-div">
+          <h4 class="company"><a href=#>Ringover</a></h4>
+        </div>
+        <div class="funding-div">
+          <h3 class="funding"><a href=#>$12M</a></h3>
+          <span class="round"><a href=#>SERIES A</a></span>
+        </div>
+        <div class="industry-div">
```
Comment (bini-i) [inline index.html:155]: Please make sure each category items have a link ```a```  tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+      <img class="latest-img" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1198933202947203078%2FyGxNiAdA.png&signature=d0c4ef35a8e410806bed50513f59f79b" alt="SwissDeCode-logo"/>
+      <div class="latest-post">
+        <div class="company-div">
+          <h4 class="company"><a href=#>SwissDeCode</a></h4>
+        </div>
+        <div class="funding-div">
+          <h3 class="funding"><a href=#>$4.2M</a></h3>
+          <span class="round"><a href=#>VENTURE CAPITAL</a></span>
+        </div>
+        <div class="industry-div">
```
Comment (bini-i) [inline index.html:176]: Please make sure each category items have a link ```a```  tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+      <img class="latest-img" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F741095900181757955%2F3Y3ZeDBE.jpg&signature=0469f20c9ad141290a048b6027e509cf" alt="Klook-logo"/>
+      <div class="latest-post">
+        <div class="company-div">
+          <h4 class="company"><a href=#>Klook</a></h4>
+        </div>
+        <div class="funding-div">
+          <h3 class="funding"><a href=#>$200M</a></h3>
+          <span class="round"><a href=#>SERIES E</a></span>
+        </div>
+        <div class="industry-div">
```
Comment (bini-i) [inline index.html:197]: Please make sure each category items have a link ```a```  tag

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Requires changes :yellow_circle: 
### Hello  @Emmyn5600 and @John-Arboleda :wave: 

#### Thanks for your submission.  

### I really like your project :+1:  and it passes most of the requirements but you can improve it by applying the following missing requirements:
* [ ] make sure to use ```header``` tag for the header
* [ ] On tablet view, inside the highlights section, the first article appearing on the left should have its title bigger than the other articles.
* [ ] Inside the latest funding section, make sure the nav menu (See more, Get updates, etc.) are grouped in a `nav` tag and each one have links (`a` tag)
* [ ] Under the latest funding section make sure each articles' category items have a link ```a```  tag (please refer to inline comments below)
* [ ]  Inside footer, at the bottom, make sure the copyright paragraph has TNW logo on the left as show [here](https://user-images.githubusercontent.com/61377322/109268687-7c9ccb80-781c-11eb-89f1-1390192d4a2c.png)
* [ ] Inside the footer, please fix the nav overflowing on mobile screen as show [here](https://user-images.githubusercontent.com/61377322/109271366-79a3da00-7820-11eb-973c-15a4c8662479.png).
#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @Emmyn5600 :tada: 

**This milestone has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

You can now merge to master.
Do not forget to fill the project completion form.
Good luck with your next milestone.

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**